test/unittests/skills/test_common_play_skill.py

```diff
@@ -0,0 +1,140 @@
+import unittest
+from unittest import mock
+
+from mycroft.messagebus import Message
+from mycroft.skills.common_play_skill import CommonPlaySkill, CPSMatchLevel
+from mycroft.skills.audioservice import AudioService
+
+
+class AnyCallable:
+    """Class matching any callable.
+
+    Useful for assert_called_with arguments.
+    """
+    def __eq__(self, other):
+        return callable(other)
+
+
+class TestCommonPlay(unittest.TestCase):
+    def test_lifecycle(self):
+        skill = CPSTest()
+        bus = mock.Mock(name='bus')
```

Comment:
Use a setup method here that executes these two lines of code.  Make skill and bus instance attributes